fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1851,6 +1848,14 @@ private void setCanRead(boolean hasLog, boolean err) {
                 } else {
                     canRead.set(true);
                     isReady.set(true);
+
```

Comment:
I think better not add the process logic here, you can add the logic in the GlobalStateMgr.waitForReady();

Reply:
good point, fixed